### PR TITLE
template updates

### PR DESCRIPTION
**Pull request by kalexand-rh**
Simplifying the PR template and moving more details to the contrib guide.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@bergerhoffer PTAL?

File: contributing_to_docs/create_or_edit_content.adoc [hunk 1]
```diff
@@ -168,11 +165,32 @@ any further changes that are required, and finally merge your work.
...
+** For community authors: Request a review by tagging @openshift/team-documentation or @kalexand-rh in a GitHub comment.
+** For Red Hat associates:
+*** For normal peer requests, add a comment that contains this text: /label peer-review-needed
+*** For normal merge review requests, add a comment that contains this text: /label merge-review-needed
+*** For urgent peer review requests, ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
+**** A link to the PR.
+**** The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
+**** Details about how the PR is urgent.
+*** For urgent merge requests, ping @merge-review-squad in the #forum-docs-review channel (CoreOS Slack workspace).
+*** Except for changes that do not impact the meaning of the content, QE review is required before content is merged.
```
Comment (bergerhoffer) [inline contributing_to_docs/create_or_edit_content.adoc:193]: You did already mention that QE is required in step 3, not sure if you wanted to repeat it again here or not

**Review by bergerhoffer — COMMENTED** [review]
This looks great! Love that the template itself is much more concise and clean, thank you!

Just added one question, but LGTM!

File: contributing_to_docs/create_or_edit_content.adoc (hunk 1 above)
Comment (kalexand-rh) [reply contributing_to_docs/create_or_edit_content.adoc:193]: I think it was mentioned twice in the original content. The context is different enough (and the statement is important enough) that I think it's ok to say twice.

Comment (bergerhoffer) [reply contributing_to_docs/create_or_edit_content.adoc:193]: Works for me! A reminder isn't a bad thing :wink:

**Pull request merged**